django/contrib/postgres/constraints.py

```diff
@@ -155,12 +155,15 @@ def __eq__(self, other):
         return super().__eq__(other)
 
     def __repr__(self):
-        return '<%s: index_type=%s, expressions=%s%s%s%s%s>' % (
+        return "<%s: index_type='%s' name=%r expressions=%s%s%s%s%s>" % (
             self.__class__.__qualname__,
             self.index_type,
-            self.expressions,
-            '' if self.condition is None else ', condition=%s' % self.condition,
-            '' if self.deferrable is None else ', deferrable=%s' % self.deferrable,
-            '' if not self.include else ', include=%s' % repr(self.include),
-            '' if not self.opclasses else ', opclasses=%s' % repr(self.opclasses),
+            self.name,
+            "'%s'" % ', '.join([
+                str(expression) for expression in self.expressions
+            ]),
+            '' if self.condition is None else " condition='%s'" % self.condition,
+            '' if self.deferrable is None else " deferrable='%s'" % self.deferrable,
```

Comment:
I would revert these changes, a string representation of `condition` and `deferrable` doesn't need and extra quotes.

Reply:
Felt a bit weird to leave those unquoted as all other arguments was quoted but it makes a better fit with your proposal of running repr on sequences (much better). 👍